train.py

```diff
@@ -1,72 +1,86 @@
 import cv2
 import numpy
+import sys
 
-from lib.utils import get_image_paths, load_images, stack_images
-from lib.training_data import get_training_data
+from utils import get_image_paths, load_images, stack_images
+from training_data import get_training_data
 
-from lib.model import autoencoder_A
-from lib.model import autoencoder_B
-from lib.model import encoder, decoder_A, decoder_B
+from model import autoencoder_A
+from model import autoencoder_B
+from model import encoder, decoder_A, decoder_B
+from getopt import getopt
+
+
+try:
+    opts, args = getopt(sys.argv[1:], "a:b:", ["image-a=", "image-b="])
+except GetoptError as err:
+    print(err)
+    sys.exit(2)
+for option, value in opts:
+    if option in ("-a", "--image-a"):
+        imageAFolder = value
+    elif option in ("-b", "--image-b"):
+        imageBFolder = value
+    else:
+        assert False, "Unhandled option"
+
+print("Image A path is {}".format(imageAFolder))
+print("Image B path is {}".format(imageBFolder))
 
 try:
-    encoder.load_weights('models/encoder.h5')
-    decoder_A.load_weights('models/decoder_A.h5')
-    decoder_B.load_weights('models/decoder_B.h5')
+    encoder.load_weights( "models/encoder.h5"   )

```

Comment:
Codestyle: Try to avoid placing spaces immediately inside parentheses, brackets or braces. https://www.python.org/dev/peps/pep-0008/#pet-peeves